client/www/scripts/modules/common/lb-build.js

```diff
@@ -424,43 +424,6 @@ module.factory(
 
         /**
          * @ngdoc method
-         * @name BuildDeployAPI.Build#createChangeStream
```

Comment:
this file shouldn't be here.

you need to merge with master to get the lates version of this file